azure-iot-device/tests/common/pipeline/test_pipeline_stages_mqtt.py

```diff
@@ -181,14 +144,14 @@ def op_set_sas_token(callback):
     return pipeline_ops_base.SetSasToken(sas_token=fake_sas_token, callback=callback)
 
 
-@pytest.mark.describe("MQTT Provider pipeline stage _runOp function with SetSasToken operations")
+@pytest.mark.describe("Provider - .run_op() -- called with SetSasToken")
```

Comment:
In a previous comment I mentioned that maybe stages/operations/etc. should have what they are, in their classname so it's clearer to understand what they are when deprived of context.

But even if we don't go down that route, it might be nice to modify some of these test names to be clearer - It took me a while to figure out what this test was for.